Resources/Lang/en_US.json

```diff
@@ -887,19 +887,19 @@
-  "FamineInfoLong": "(Apocalypse): \nOnce the Baker has the set amount of people with bread alive, they will become Famine. If Famine is not voted out after the meeting they become Famine, every player without bread will starve (excluding other Apocalypse members).\nAfter this starvation of everyone without bread, Famine can use their kill button to starve any remaining players, which will kill those players right before the next meeting.\n\n<b>You are invincible and your presence is announced to everyone at the meeting after you transform.</b>",
-  "BerserkerInfoLong": "(Apocalypse):\nAs the Berserker, you level up with each kill.\nUpon reaching a certain level defined by the host, you unlock a new power.\n\nScavenged kills make your kills disappear.\nBombed kills make your kills explode. Be careful when killing, as this can kill your other Apocalypse members if they are near. \nAfter a certain level you become War.",
+  "DeathInfoLong": "(Apocalypse): \nOnce the Soul Collector has collected their needed souls, they become Death. Death kills everyone and wins if Death is not ejected by the end of the next meeting.\nA configurable amount of extra meeting time will be given on the meeting Death transforms to have more discussion to find Death.\n\n<b>You are invincible, and your presence is announced to everyone at the meeting after you transform.</b>",
+  "BakerInfoLong": "(Apocalypse): \nAs the Baker, you can use your kill button on a player per round to give them bread. \nOnce a set amount of players are alive with bread, you become Famine.\n\nIf the Bread gives additional effects and the setting is on, then you can vent to change the bread that you give out. \n<b>Bread Effects:</b>\nReveal: Reveals the target's role to the Baker (stays the whole game)\nRoleblock: Sets the target's kill cooldown to 999 (resets to normal after the meeting)\nBarrier: Gives the target a barrier that is only known to the Baker (barrier is removed after the meeting)",
+  "FamineInfoLong": "(Apocalypse): \nOnce the Baker has the set amount of people with bread alive, they will become Famine. Suppose Famine is not voted out after the meeting. Then they become Famine, and every player without bread will starve (excluding other Apocalypse members).\nAfter this starvation of everyone without bread, Famine can use their kill button to starve any remaining players, which will kill those players right before the next meeting.\n\n<b>You are invincible, and your presence is announced to everyone at the meeting after you transform.</b>",
```

Comment:
This seems wrong to the tongue. Why are you saying "Suppose"? You can reword this significantly better.